docs/intro/formats.md

```diff
@@ -1,76 +1,79 @@
-  read these files, the executable *vpd2vcd* must be in your \$PATH
-  during *configure* and *gtkwave* must be invoked with the -o option.
-- WLF: Wave Log File. This is generated by ModelSim. In order to read
-  these files, the executable *wlf2vcd* must be in your \$PATH during
-  *configure* and *gtkwave* must be invoked with the -o option.
-- FSDB: Fast Signal Database. Reading these files generally requires
-  that the executables *fsdb2vcd* and *fsdbdebug* are in your \$PATH
-  during *configure* and *gtkwave* must be invoked with the -o option.
-  FSDB files can also be read without conversion with a processing
-  speed similar to FST if the FsdbReader libraries nffr and nsys are
-  found during *configure*, pointed to by the environment variable
-  FSDBREADER_LIBS. Headers are pointed to by FSDBREADER_HDRS.
-
-Converter helper applications are packaged with the viewer in order to
-convert VCD files into LXT, LXT2, VZT, or FST files. Conversion from
-LXT2, VZT, and FST back into VCD is possible. Wholesale conversion from
-LXT is not currently possible, however it is possible to save the traces
-visible in the main GTKWave window as VCD so conversion to LXT is not
-strictly irreversible.
+# Supported formats
+
+GTKWave supports various file formats.
+
+- `VCD`: Value Change Dump. This is an industry standard file format
+  generated by most Verilog simulators and is specified in IEEE-1364.
+  This is the slowest of the formats for the viewer to process and
+  requires the most memory. However, this format is ubiquitous and
+  almost all tools support it, which is why native support remains.
+  Note that recent versions of the viewer default to dynamic VCD
+  recoding in memory through some interesting tricks with zlib
+  compressed VLists. (See [VCD Recoding](../vcd-recoding.md))
+  This greatly reduces the amount of memory
+  required to store a large, full (non-interactive) VCD trace in
+  memory such that in many cases, less memory is required than the
+  actual size of the trace itself. Nevertheless, using one of the
+  database formats will almost always be more efficient for larger
+  traces, especially if they are to be viewed repeatedly. (i.e., the
+  speed hit for converting a trace to a database format is offset by
+  the repeated cost of recoding VCD every time the trace is viewed.)
+  The more physical memory that is available on a machine being used
+  to view VCD, the better.
+- `FST`: Fast Signal Trace. It is a trace file format developed by GTKWave
+  which is designed for very fast sequential and random access.
+- `LXT`: InterLaced eXtensible Trace. This is an optimized format
+  utilizing interleaved back pointers and value changes. Processing
+  LXT files is faster than VCD. It was created specifically for use
+  with GTKWave, however some other simulators (notably, Icarus
+  Verilog) support it natively. **Deprecated. Will be removed in
+  GTKWave 4.**
```

Comment:
I think it would be easier to read if we mention all depreciated formats in a single attention block.